tests/Metadata/Property/Factory/InheritedPropertyNameCollectionFactoryTest.php

```diff
@@ -27,18 +27,33 @@
  */
 class InheritedPropertyNameCollectionFactoryTest extends TestCase
 {
-    public function testCreate()
+    public function testCreateOnParent()
     {
         $resourceNameCollectionFactory = $this->prophesize(ResourceNameCollectionFactoryInterface::class);
         $resourceNameCollectionFactory->create()->willReturn(new ResourceNameCollection([DummyTableInheritance::class, DummyTableInheritanceChild::class]))->shouldBeCalled();
 
         $propertyNameCollectionFactory = $this->prophesize(PropertyNameCollectionFactoryInterface::class);
         $propertyNameCollectionFactory->create(DummyTableInheritance::class, [])->willReturn(new PropertyNameCollection(['name']))->shouldBeCalled();
-        $propertyNameCollectionFactory->create(DummyTableInheritanceChild::class, [])->willReturn(new PropertyNameCollection(['nickname', '169']))->shouldBeCalled();
+        $propertyNameCollectionFactory->create(DummyTableInheritanceChild::class, [])->willReturn(new PropertyNameCollection(['nickname', '169']));
```

Comment:
`->shouldNotBeCalled` then?

Reply:
I guess, willReturn is probably unnecessary too. Ill make the change